C/CyrusSASL/build_tarballs.jl

```diff
@@ -3,11 +3,12 @@
 using BinaryBuilder, Pkg
 
 name = "CyrusSASL"
-version = v"2.1.28"
+version = v"2.1.28+openssl3"
```

Comment:
You can't do this, you must use a different X.Y.Z version number, build numbers aren't acceptable.